### ISSUE TITLE
does not build on latest rustc

### ISSUE THREAD

**Issue by aentity** [maxmarsc/audeye#3]
```
error[E0283]: type annotations needed
  --> src/render/renderer.rs:84:10
   |
84 |         .constraints(constraints.as_ref())
   |          ^^^^^^^^^^^             ------ type must be known at this point
   |          |
   |          cannot infer type of the type parameter `C` declared on the method `constraints`
   |
   = note: multiple `impl`s satisfying `Vec<Constraint>: AsRef<_>` found in the `alloc` crate:
           - impl<T, A> AsRef<Vec<T, A>> for Vec<T, A>
             where A: Allocator;
           - impl<T, A> AsRef<[T]> for Vec<T, A>
             where A: Allocator;
help: consider specifying the generic argument
   |
84 |         .constraints::<&T>(constraints.as_ref())
   |                     ++++++
```
On rustc 1.70.0 it builds

**Comment by maxmarsc** [comment]
Is latest the `1.75.0` ? Sorry I've not touched rust in a while, I'll look into it :)

**Comment by aentity** [comment]
1.76.0 is the latest :)

**Comment by maxmarsc** [comment]
Fixed on v0.2.1
